Sources/CoreLocationMiddleware/CoreLocationMiddleware.swift

```diff
@@ -1,40 +1,59 @@
 import CoreLocation
 import SwiftRex
 
-public struct LocationState: Equatable {
-    var authzType: AuthzType
-    var authzStatus: CLAuthorizationStatus
-    var location: CLLocation
-    
-    public init(authzType: AuthzType = .whenInUse,
-                authzStatus: CLAuthorizationStatus,
-                location: CLLocation) {
-        self.authzType = authzType
-        self.authzStatus = authzStatus
-        self.location = location
-    }
-}
 
+// MARK: - ACTION
 public enum LocationAction {
     // Input
+    case request(RequestAction)
+    // Output
+    case status(StatusAction)
+}
+
+public enum RequestAction {
     case startMonitoring
     case stopMonitoring
     case requestAuthorizationStatus
     case requestAuthorizationType
     case requestPosition
     case requestDeviceCapabilities
-    // Output
+}
+
+public enum StatusAction {
     case gotPosition(CLLocation)
-    case gotAuthzStatus(CLAuthorizationStatus)
+    case gotAuthzStatus(AuthzStatus)
```

Comment:
This is the important part for the change specific to sending the CLAccuracyAuthorization value.